mediorum/main.go

```diff
@@ -64,7 +66,8 @@ func startStagingOrProd(isProd bool) {
 
 	// use custom DNS in staging + prod
 	// could scope it down to a TLD, but first want to test it works everywhere without surprises
-	httputil.UseCustomDNS()
+	// httputil.UseCustomDNS()
+	// TODO: custom DNS makes mediorum unable to communicate with the radix container
```

Comment:
verifying these TODOs are meant to be addressed before this goes out to prod

Reply:
the other TODOs are fine to wait. i want to ask what @stereosteve thinks of leaving this commented out or removing since it didn't fix staked's network issues?

Reply:
Yeah if it didn't fix the staked issue we can delete it